src/notifications/ios_service.md

```diff
@@ -0,0 +1,57 @@
+# Setting up the Notification Service Extension
+
+## Add a Notfication Service Extension target
+
+In Xcode add a new Target to your application:
+1. In the menu `File > New > Target...`
+2. Select `Notification Service Extension`
+3. Give the name `NotificationService`
+
+This should create a new target with a bundle identifier like `com.example.application.NotificationService` and a folder in your application called `NotificationService`.
+
+## Identifiers
+
+To share data between the main application and service extension targets, you need to add a common `app group` to the main application identifier and new service extension identifier.  Go the [Apple Developer](https://developer.apple.com/account/resources/identifiers/list/applicationGroup) portal, in the Identifiers section select the App Groups dropdown to the right, then:
+1. Click `+` to add an App Group
+2. Select `App Groups` from the list
+3. Give the App Group a description and identifier, like `group.com.example.application`
```

Comment:
I think it's better to use a more formal language such as `such as` instead of "like <-example->" on this page :)